### PR TITLE
#4 demo graph created

### PR DESCRIPTION
**Pull request by oskarjerzyk**
## Description
The purpose of this task is to create a demo scenario which will allow us to test graph visualization in chrome dev tools. This task is related to [this issue](https://github.com/Knotx/knotx-fragments-chrome-extension/issues/4)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: src/functionalTest/resources/content/paymentsWithHandlebars.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title><!--
```
Comment (Voycawojka) [inline src/functionalTest/resources/content/paymentsWithHandlebars.html]: This title tag shouldn't be here

File: src/functionalTest/resources/conf/knots/pebbleTemplateEngineKnot.conf [hunk 2]
```diff
@@ -16,6 +16,7 @@ engine {
 
     syntax = {
       # strictVariables = true
+      wrappingRootNodeName = root
```
Comment (tomaszmichalak) [inline src/functionalTest/resources/conf/knots/pebbleTemplateEngineKnot.conf]: This value is empty by default. Can we configure it for cases where required?

File: src/functionalTest/resources/scenarios/pebble-template-engine/pebble.conf [hunk 3]
```diff
@@ -0,0 +1 @@
+config.pebbleTemplateEngine.options.config.engine.config.syntax.wrappingRootNodeName = ""
```
Comment (tomaszmichalak) [inline src/functionalTest/resources/scenarios/pebble-template-engine/pebble.conf]: I expect that it should be configured only where required.

File: src/functionalTest/java/io/knotx/stack/functional/KnotxFragmentsDebugDataWithHandlebarsTest.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+          assertEquals(HttpResponseStatus.OK.code(), resp.statusCode());
+          String response = resp.bodyAsString();
+          assertNotNull(response);
+
+          String scriptRegexp = "<script data-knotx-id=\"?.*?\" type=\"application/json\">(?<fragmentEventJson>.*?)</script>";
+          Pattern scriptPattern = Pattern.compile(scriptRegexp, Pattern.DOTALL);
+          Matcher matcher = scriptPattern.matcher(response);
+
+          //contains 2 fragments
+          assertTrue(matcher.find());
```
Comment (tomaszmichalak) [inline src/functionalTest/java/io/knotx/stack/functional/KnotxFragmentsDebugDataWithHandlebarsTest.java]: Can you please try to validate node log too?

File: src/functionalTest/resources/content/paymentsWithHandlebars.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```
Comment (tomaszmichalak) [inline src/functionalTest/resources/content/paymentsWithHandlebars.html]: Do we need this test?

File: src/functionalTest/resources/service/mock/placeholderFallback.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
+  "fallback": "fallback"
```
Comment (tomaszmichalak) [inline src/functionalTest/resources/service/mock/placeholderFallback.json]: Maybe some more verbose json:
```
{
  "id": 999
  "method": "Default",
  "price": "$10"
}
```

**Pull request merged**